src/main.py

```diff
@@ -0,0 +1,52 @@
+import os
+import psutil
+from fastapi import FastAPI, APIRouter, Depends, status
+from functools import lru_cache
+from .config import Settings
+from .osw_validator import OswValidator
+
+app = FastAPI()
+
+prefix_router = APIRouter(prefix='/health')
+
+
+@lru_cache()
+def get_settings():
+    return Settings()
+
+
+@app.on_event('startup')
+async def startup_event(settings: Settings = Depends(get_settings)) -> None:
+    try:
+        OswValidator().start_listening()
+    except:
+        print('\n\n\x1b[31m Application startup failed due to missing or invalid .env file \x1b[0m')
+        print('\x1b[31m Please provide the valid .env file and .env file should contains following parameters\x1b[0m')
+        print()
+        print('\x1b[31m UPLOAD_TOPIC=xxxx \x1b[0m')
+        print('\x1b[31m UPLOAD_SUBSCRIPTION=xxxx \x1b[0m')
+        print('\x1b[31m VALIDATION_TOPIC=xxxx \x1b[0m')
+        print('\x1b[31m QUEUECONNECTION=xxxx \x1b[0m')
```

Comment:
Do not print the connection strings. They are supposed to be private

Reply:
@susrisha  this is not about printing connection string. This would be shown on if - 

1. Missing .env file
2. Invalid .env file

We are closing the server by saying 

```
Application startup failed due to missing or invalid .env 
Please provide the valid .env file and .env file should contains following parameters


UPLOAD_TOPIC=xxxx
UPLOAD_SUBSCRIPTION=xxxx
VALIDATION_TOPIC=xxxx
QUEUECONNECTION=xxxx
STORAGECONNECTION=xxxx


```